How to Conduct a Successful Interview.md

```diff
@@ -0,0 +1,39 @@
+# How to Conduct a Successful Interview
+
+When writing a blog post or article, sometimes it’s useful to include an expert opinion or personal quote to boost your credibility and to improve engagement with your content. If you’ve been assigned to conduct an interview or if you think it could benefit your writing task, follow these guidelines to successfully conduct a professional interview.
+
+- **Ask, don’t expect.**
+
+Generally, it’s best to never assume someone is free. In an email or a Slack message, ask the person politely if they would be interested in answering some questions. When sending the first message, outline what you’re working on and the topic you would like to interview them about so they know what they’re agreeing to from the beginning.
+
+- **Come prepared.**
+
```

Comment:
Also for our case, being able to ask basic questions in different formats - if you can save their time by emailing/dm'ing them questions and getting written responses vs. sometimes you only want to have that in person experience